src/app/teams/teams-view.component.html

```diff
@@ -5,7 +5,7 @@
 <div class="space-container">
   <mat-toolbar class="primary-color font-size-1">
     <h3 class="margin-lr-3 ellipsis-title">{{team?.name}}</h3>
-    <button class="margin-lr-5" mat-stroked-button i18n (click)="changeLeftTile()" *ngIf="userStatus === 'member'">Show { leftTileContent, select, description {Messages} news {Description} }</button>
+    <button class="margin-lr-5" mat-stroked-button i18n (click)="changeLeftTile()" *ngIf="userStatus === 'member'">Show { leftTileContent, select, description {Messages} news {Description}}</button>
```

Comment:
It looks like this change is just one removed space?  Please avoid changes like this so when reviewing the only text highlighted in GitHub is related to the change.